java/src/jmri/jmrix/mqtt/MqttAdapter.java

```diff
@@ -86,6 +86,10 @@ public MqttAdapter() {
         options.put("13", new Option("Reporter topic :",    new String[]{Bundle.getMessage("TopicReporter")}, Option.Type.TEXT));
         options.put("14", new Option("Signal Head topic :", new String[]{Bundle.getMessage("TopicSignalHead")}, Option.Type.TEXT));
         options.put("15", new Option("Signal Mast topic :", new String[]{Bundle.getMessage("TopicSignalMast")}, Option.Type.TEXT));
+        options.put("LastWillTopic", new Option(Bundle.getMessage("NameTopicLastWill"),
```

Comment:
Lines above the new ones use numbers for the "options."  Should they be changed or should the new lines simply have a comment saying that option "numbers" can also be meaningful character strings?  The inconsistency will no doubt cause others to wonder why at some point in the future so a little explanation now might save future trouble.

"